app/controllers/application_controller.rb

```diff
@@ -5,6 +5,133 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "secret"
+  end
+get '/tweets/new' do 
+    if Helpers.is_logged_in?(session)
+      erb :'/tweets/create_tweet'
+    else 
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do
+    if params[:content] == ""
+      redirect :'/tweets/new'
+    else
+      user = User.find_by_id(session[:user_id])
+      Tweet.create(content: params[:content], user_id: user.id)
+    end
+
+  end
+
+  get '/tweets' do 
+
+    if session[:user_id]
```

Comment:
You can use your `logged_in?` helper method here, which is preferable to referring to the session object directly.